packages/react-scripts/package.json

```diff
@@ -71,6 +70,12 @@
     "postcss-normalize": "7.0.1",
     "postcss-preset-env": "6.6.0",
     "postcss-safe-parser": "4.0.1",
+    "prettier": "^1.18.2",
+    "sass-loader": "^7.1.0",
```

Comment:
Why do we need this and `style-loader` I dont think we use them in webpack config rn?

Reply:
@valerii-cognite